shell/browser/osr/osr_render_widget_host_view.cc

```diff
@@ -542,6 +542,11 @@ OffScreenRenderWidgetHostView::CreateViewForWidget(
   if (auto* rwhv = render_widget_host->GetView())
     return static_cast<content::RenderWidgetHostViewBase*>(rwhv);
 
+  if (web_contents_view) {
+    return static_cast<content::RenderWidgetHostViewBase*>(
+        web_contents_view->CreateViewForWidget(render_widget_host));
+  }
```

Comment:
why is this needed ? Seems additional to the previous logic when the patch was in place.

Reply:
reading through the source seemed to indicate to me that it could be possibly nullptr, so i figured i'd add a extra safety check while i was here